src/main/java/uk/gov/hmcts/reform/logging/layout/ReformLoggingLayout.java

```diff
@@ -71,7 +75,19 @@ public String doLayout(ILoggingEvent event) {
         log.append(event.getFormattedMessage()).append(CoreConstants.LINE_SEPARATOR);
 
         if (proxy != null) {
-            proxy.fullDump();
+            Stream<StackTraceElementProxy> trace = Arrays.stream(proxy.getStackTraceElementProxyArray());
```

Comment:
were you able to test if the caused by is present?

Reply:
did not test that but fair point to add into stacktrace. shame it doesn't have some depth while crawling through steps.